src/extension/VSCodeSessionManager.ts

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+
+import * as vscode from "vscode";
+import * as Net from "net";
+import { DEBUG_TYPES } from "./debugConfigurationProvider";
+import { RNDebugSession } from "../debugger/rnDebugSession";
+import { DebugSessionBase, TerminateEventArgs } from "../debugger/debugSessionBase";
+import { DirectDebugSession } from "../debugger/direct/directDebugSession";
+
+export class VSCodeSessionManager implements vscode.DebugAdapterDescriptorFactory, vscode.Disposable {
```

Comment:
Maybe it is better to name it ReactNativeSessionManager? What do you think?

Reply:
Yeah, sounds good, I'll rename this class.